pkg/virt-handler/heartbeat/heartbeat.go

```diff
@@ -57,9 +59,15 @@ func (h *HeartBeat) Run(heartBeatInterval time.Duration, stopCh chan struct{}) (
 		h.labelNodeUnschedulable()
 		close(done)
 	}()
+	go h.monitorMultipath.Run()
```

Comment:
I don't think we need this in the heartbeat anymore. I guess it could just run separately, similarly to the device manager (in vm.go Run())

Reply:
unless we want to make the node Unschedulable until we have an established connection to multipathd 

maybe we also need a channel into this thread to report and act on errors?

Reply:
No, I won't do this, because otherwise, it makes the multipathd a requirements and that's not. Additionally, this is used only with scsi reservation and it isn't always enabled. We could add some configuration in KubeVirt CR, but then, this isn't backportable.

Reply:
Yes, that's what I thought. I just wanted to make sure.